src/components/events/EventsSection.jsx

```diff
@@ -5,6 +5,7 @@ import Table from '@material-ui/core/Table';
 import TableBody from '@material-ui/core/TableBody';
 
 import db from '../../firebase/firebase';
+import Loading from '../interface/Loading';
```

Comment:
let's order these up, Import the interfaces in alphabetical order than we'll import EventTableRow last :)